lms/static/js/instructor_dashboard/certificates.js

```diff
@@ -0,0 +1,37 @@
+var edx = edx || {};
+
+(function( $, gettext ) {
+    'use strict';
+
+    edx.instructor_dashboard = edx.instructor_dashboard || {};
+    edx.instructor_dashboard.certificates = {};
+
+    $(function() {
+        /**
+         * Show a confirmation message before letting staff members
+         * enable/disable self-generated certificates for a course.
+         */
+        $('#enable-certificates-form').on('submit', function( event ) {
+            var isEnabled = $('#certificates-enabled').val() === 'true',
+                confirmMessage = '';
+
+            if ( isEnabled ) {
+                confirmMessage = gettext('Allow students to generate certificates for this course?');
+            } else {
+                confirmMessage = gettext('Prevent students from generating certificates in this course?');
```

Comment:
@srpearce These strings need a doc review.  They appear in the confirmation dialog when a PM clicks "enable student-generated certificates" or "disable student-generated certificates".